### ISSUE TITLE
Helm v3 - Linting multiple charts merges the error messages together

### ISSUE THREAD

**Issue by andytom** [helm/helm#6247]
Output of `helm version`:
```
$ helm3 version
version.BuildInfo{Version:"v3.0.0-alpha.2", GitCommit:"97e7461e41455e58d89b4d7d192fed5352001d44", GitTreeState:"clean", GoVersion:"go1.12.7"}
```

Output of `kubectl version`:
N/A

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
N/A

When linting multiple charts in helm3 all error messages are merged together making it hard to work out which chart caused a failure.

```
$ mkdir -p /tmp/broken_chart_{1,2}
$ touch /tmp/broken_chart_1/Chart.yaml
$ helm3 lint /tmp/broken_chart*
Error: 1 chart(s) linted, 2 chart(s) failed
        %!s(<nil>)
        no chart found for linting (missing Chart.yaml)
        [ERROR] Chart.yaml: name is required
        [ERROR] Chart.yaml: directory name (broken_chart_1) and chart name () must be the same
        [ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"
        [ERROR] Chart.yaml: version is required
        [INFO] Chart.yaml: icon is recommended
        [INFO] values.yaml: file does not exist
        [WARNING] templates/: directory not found 
```

I'm not sure if this is the expected behavior of the lint action or not but this change makes it difficult to work out what failed compared to the same command in helm2.
```
$ helm lint /tmp/broken_chart*
==> Linting /tmp/broken_chart_1
[ERROR] Chart.yaml: name is required
[ERROR] Chart.yaml: directory name (broken_chart_1) and chart name () must be the same
[ERROR] Chart.yaml: version is required
[INFO] Chart.yaml: icon is recommended
[INFO] values.yaml: file does not exist
[WARNING] templates/: directory not found

==> Skipping /tmp/broken_chart_2
No chart found for linting (missing Chart.yaml)

Error: 1 chart(s) linted, 2 chart(s) failed
```

**Comment by mattfarina** [comment]
I think this is a UX bug. PRs welcome to fix.

**Comment by andytom** [comment]
Have done some initial work on this in https://github.com/helm/helm/pull/6296, will continue to work on this.

**Comment by andytom** [comment]
@mattfarina I've taken a shot at this and opened two PRs #6296 and #6310.

**Comment by andytom** [comment]
Closing this as fixed by #6310 and #6296.
